app/commands/decidim/vocdoni/admin/update_election.rb

```diff
@@ -55,6 +55,18 @@ def update_election!
             visibility: "all"
           )
         end
+
+        def election_type_attributes
+          {
+            election_type: {
+              auto_start: form.auto_start,
+              interruptible: form.interruptible,
+              dynamic_census: form.dynamic_census,
+              secret_until_the_end: form.secret_until_the_end,
+              anonymous: form.anonymous
```

Comment:
Some of these configurations are not in the frontend, I supose they will be in the future?

Reply:
Yes, at least `interruptible` for sure, as that's part of the grant and we need to support that. I'm working on this one on another PR.

See #35: 

> v. Manage votes on-demand according to the Election lifecycle states specifications as defined in [developer.vocdoni.io/get-started/intro#election-lifecycle-states](https://developer.vocdoni.io/get-started/intro#election-lifecycle-states)

The others are supported on the API, but at least some are not (yet) implemented in the SDK. I've got them from https://github.com/vocdoni/vocdoni-sdk#creating-a-voting-process

Reply:
great, let's leave it like this